includes/glm/gtx/extended_min_max.hpp

```diff
@@ -171,7 +171,7 @@ namespace glm
 	/// @tparam T Floating-point scalar types
 	/// @tparam Q Value from qualifier enum
 	///
-	/// @see gtx_extented_min_max
+	/// @see gtx_extended_min_max
 	template<length_t L, typename T, qualifier Q>
 	GLM_FUNC_DECL vec<L, T, Q> fclamp(vec<L, T, Q> const& x, vec<L, T, Q> const& minVal, vec<L, T, Q> const& maxVal);
 
```

Comment:
Hanging a comment to note a renamed file below...

Not entirely certain about it...